Backend/ServerManager.py

```diff
@@ -343,6 +343,9 @@ def api_get_economic_data():
         traceback.print_exc()
         return 'ERROR ENDPOINT ' + errStr
 
+@app.route('/wordcloud', methods=['GET'])
+def api_get_wordcloud():
+    return send_file(AlgorithmsManager.create_wordcloud_image(), mimetype='image/png')
```

Comment:
are you sure that this will work?
This source says "By default, the only place that Flask will look for files within your app will be the static directory"
https://pythonprogramming.net/flask-protected-files-tutorial/
I think you might want to store the wordcloud files in some new directory like "temp" (which would also be nice for cleanup, since otherwise these generated files would pile up and badly clutter the server's main directory) and then use the function send_from_directory("temp", some_filename_within_temp) instead

Reply:
Okay!

Reply:
Oops, since this is working it seems I misunderstood that documentation :(